Bottom: 88938e167fd1210ae4933a4f9d29825bf3025425
Top:    5faf4c48229ca1e412332f443ce006fb33a0caa3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-06-05 12:07:57 +0100

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 3e57696..d6d6273 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -576,6 +576,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
